MainApp/models.py

```diff
@@ -25,7 +25,11 @@ def visible_info(self):
         if not self.tweet_text:
             return
         return self.tweet_text
-    
+
+    def countLikes(self):
+        return len(Like.objects.filter(tweet=self))
+
+    likes = property(countLikes)    
```

Comment:
This added a property to get the number of likes to the current tweet.
Whenever we are referencing a tweet object in a django template we can just use tweet.likes to get the number of likes